### ISSUE TITLE
schedule.add parameter can not use “after“

### ISSUE THREAD

**Issue by huangfupeng** [saltstack/salt#39710]
### Description of Issue/Question
when I use  schedule.add add  schedule with parameter ''after'
I can find after in the schedule
I run "salt 'my-minion' schedule.add job4 function='cmd.run' job_args="['date >> /tmp/cc.log']"  after='02/27/2017 02:15pm' seconds=5"
but I can find "after"
### Setup
(Please provide relevant configs and/or SLS files (Be sure to remove sensitive info).)
`salt 'my-minion' schedule.add job4 function='cmd.run' job_args="['date >> /tmp/cc.log']"  after='02/27/2017 02:15pm' seconds=5`

```
salt 'my-minion' schedule.list

 job4:
        args:
        - date >> /tmp/cc.log
        enabled: true
        function: cmd.run
        jid_include: true
        maxrunning: 1
        name: job4
        seconds: 5

```

### Steps to Reproduce Issue
(Include debug logs if possible and relevant.)
I find file  “salt/modules/schedule.py“  362

I can not see 'after' in the list
```

    if 'splay' in kwargs:
        if isinstance(kwargs['splay'], dict):
            # Ensure ordering of start and end arguments
            schedule[name]['splay'] = salt.utils.odict.OrderedDict()
            schedule[name]['splay']['start'] = kwargs['splay']['start']
            schedule[name]['splay']['end'] = kwargs['splay']['end']
        else:
            schedule[name]['splay'] = kwargs['splay']

    for item in ['range', 'when', 'once', 'once_fmt', 'cron', 'returner',
                 'return_config', 'return_kwargs', 'until', 'run_on_start']:
        if item in kwargs:
            schedule[name][item] = kwargs[item]

    return schedule[name]

```
### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
Salt Version:
           Salt: 2016.11.2

**Comment by garethgreenaway** [comment]
Taking a look.

**Comment by Ch3LL** [comment]
@huangfupeng can you give the fix in #39734 a try?
